src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -119,7 +119,8 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       break;
 
     case VenueTemplate.conversationspace:
-      template = <ConversationSpace />;
+      template = <ConversationSpace venue={venue} />;
+      hasBackButton = false;
```

Comment:
```suggestion
      // Remove the back button, because we don't need it in Table view
      hasBackButton = false;
```

Reply:
fixed in c31874f2e06bf3af652db14f326f77495653a166

Reply:
fixed in 088c807569a37b13ce953e79d623431827421b6a

Reply:
accidentally added this twice, but now it's fixed (one comment exists)